backend/api/Services/QuestionTemplateService.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+using HotChocolate;
+
+using api.Context;
+using api.Models;
+
+namespace api.Services
+{
+    public class QuestionTemplateService
+    {
+        private readonly BmtDbContext _context;
+
+        public QuestionTemplateService([Service] BmtDbContext context)
+        {
+            _context = context;
+        }
+
+        public List<QuestionTemplate> ActiveQuestions()
+        {
+            List<QuestionTemplate> questions = new List<QuestionTemplate>();
+            foreach (QuestionTemplate template in _context.QuestionTemplates)
+            {
+                if (template.Status.Equals(Status.Active))
+                {
+                    questions.Add(template);
+                }
+            }
```

Comment:
I think it might be better in the long run to use the database for filtering. We should maybe avoid for each loops where we can. 

```suggestion
            List<QuestionTemplate> questions = _context.QuestionTemplates.Where(template => template.Status.Equals(Status.Active));
```

Reply:
yes agreed